src/test/cases/command_parameters/ancestry.json

```diff
@@ -0,0 +1,17 @@
+{
+  "param_test": [""],
+  "param_test.a": ["param_test", ""],
+  "param_test.b": ["param_test", ""],
+  "param_test.c": ["param_test", ""],
+  "param_test.d": ["param_test", ""],
+  "param_test.e": ["param_test", ""],
+  "param_test.f": ["param_test", ""],
+  "wf": [""],
+  "wf.param_test": ["wf", ""],
+  "wf.param_test.a": ["wf.param_test", "wf", ""],
+  "wf.param_test.b": ["wf.param_test", "wf", ""],
+  "wf.param_test.c": ["wf.param_test", "wf", ""],
+  "wf.param_test.d": ["wf.param_test", "wf", ""],
+  "wf.param_test.e": ["wf.param_test", "wf", ""],
+  "wf.param_test.f": ["wf.param_test", "wf", ""]
+}
```

Comment:
The new PR now no longer has Scott's changes incorporated. Had to make a separate PR.